src/partials/partial.erb.html

```diff
@@ -1,4 +1,4 @@
-<div data-key="lmn-book" data-name="<%= customisation.name %>"
+<div data-icons="true" data-key="lmn-book" data-name="<%= customisation.name %>"
```

Comment:
I was getting confused here. Using the monkey wrapper in eagle, I will have to add it as a data attribute for reasons.